src/e2e-browser/globalSetup.ts

```diff
@@ -19,29 +19,31 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { t, ClientFunction, Selector } from "testcafe";
-import { screen } from "@testing-library/testcafe";
+import { spawn } from "child_process";
```

Comment:
`globalSetup.ts` has nothing to do with `gluu.ts` (which I didn't port since Gluu was removed). It basically starts the sandbox server.